src/pages/chats/messages-window/messages-window.component.ts

```diff
@@ -0,0 +1,47 @@
+import {Block} from '../../../components/block';
+import {MessageComponent} from './message/message.component';
+
+const defaultTemplate = '.plug Выберите чат чтобы отправить сообщение';
+
+const template = `.main-content-wrapper
+    .header
+        .wrapper
+            .img-plug
+            span.user-name #{selectedChat.name}
+            .options-icon
+    .messages-wrapper
+        .messages-container
+             each msg in messages
+                != msg
+    .footer-panel
+        form
+            .attachments-icon
+            input.message-input(id='messageInput' type='text', placeholder='Сообщение', name='message')
+            .send-icon.disabled(id='sendIcon')`;
+
+export class MessagesWindowComponent extends Block {
+  messageComponents: MessageComponent[] = [];
+
+  constructor(props) {
+    super('section', props);
+  }
+
+  render() {
+    if (this.props.selectedChat) {
+      return this.compile(template, this.props);
+    }
+
+    return this.compile(defaultTemplate, this.props);
+  }
+
+  addEvents() {
+    document.getElementById('messageInput').addEventListener('input', event => {
```

Comment:
Правильней искать в компоненте а не в документе, мало ли сколько будет таких инпутов в документе, а нас интересует конкретный (этого компонента)
```suggestion
    this.getContent().getElementById('messageInput').addEventListener('input', event => {
```

Reply:
Реализовал так везде